sdk/tests/conformance/more/unit.js

```diff
@@ -71,6 +71,44 @@ var __testFailCount__ = 0;
 var __testLog__;
 var __backlog__ = [];
 
+var getUrlOptions = (function() {
```

Comment:
Would it be possible to refactor these more/ tests to at least share this code with js-test-pre.js instead of introducing two copies into the source base?